capitalprojects_build/11_spending.sh

```diff
@@ -1,6 +1,102 @@
 #!/bin/bash
-if [ -f .env ]
-then
-  export $(cat .env | sed 's/#.*//g' | xargs)
-fi
-python3 python/get_checkbook.py
\ No newline at end of file
+source config.sh
+
+# Import fisa_capitalcommitments to database
+version=$(get_version fisa_capitalcommitments)
+location=US
+
+function import {
+    echo "version: $version"
+    import_private fisa_capitalcommitments $version
+}
+
+# Loading the latest fisa_capitalcommitments to bigquery
+function fisa {
+    mkdir -p .output && (
+        cd .output
+        local dataset=fisa_capitalcommitments
+        local tablename=$dataset.$version
+        echo "$dataset"
+        CSV_export $dataset
+        bq mk \
+            --location=$location\
+            --dataset $dataset
+        bq mk $tablename
```

Comment:
Do we need to include `--replace` flag? See the errors [here](https://github.com/NYCPlanning/db-cpdb/runs/2668341719?check_suite_focus=true#step:6:65). From what I can tell from the docs, that flag is designed for making a table from a query.  I'm not sure how to do the bq equivalent of `CREATE IF NOT EXISTS`.

Reply:
![image](https://user-images.githubusercontent.com/13207770/119558291-37326080-bd6f-11eb-828a-9933a4c19f73.png)

Reply:
I think we might not need `bq mk $tablename`, but I guess it doesn't hurt to have it